src/test/java/uk/sky/cqlmigrate/CqlMigratorImplTest.java

```diff
@@ -241,10 +248,10 @@ public void shouldThrowExceptionWhenMultipleBootstrap() throws Exception {
         Collection<Path> cqlPaths = asList(getResourcePath("cql_bootstrap"), getResourcePath("cql_bootstrap_duplicate"));
 
         //when
-        MIGRATOR.migrate(CASSANDRA_HOSTS, binaryPort, username, password, TEST_KEYSPACE, cqlPaths);
+        MIGRATOR.migrate(CASSANDRA_HOSTS, LOCAL_DC, binaryPort, username, password, TEST_KEYSPACE, cqlPaths);
 
         //then
-        cluster.connect(TEST_KEYSPACE);
+        //throws exception
```

Comment:
Could we catch the throwable and assert it's type instead of nothing? This has highlighted that it's not obvious which functional call the exception was previously being thrown from